governance.md

```diff
@@ -7,7 +7,7 @@ Maintainers MUST remain active. If they are unresponsive for >3 months, they wil
 
 New maintainers can be added to the project by a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote of the existing maintainers. A potential maintainer may be nominated by an existing maintainer. A vote is conducted in private between the current maintainers over the course of a one week voting period. At the end of the week, votes are counted and a pull request is made on the repo adding the new maintainer to the [OWNERS](OWNERS.md) file.
 
-A maintainer may step down by submitting an [issue](https://github.com/deislabs/cnab-spec/issues/new) stating their intent.
+A maintainer may step down by submitting an [issue](https://github.com/oam-dev/rudr/issues) stating their intent.
```

Comment:
That's still not right, should be oam-dev/spec/issues

Reply:
Resolved this now.